### ISSUE TITLE
Fix a11y issue with sandbox button

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-org#1534]
The new sandbox buttons added to the component documentation need discernible text for screen readers. Screen reader users are not able to discern the purpose of button that has no accessible name.

The button should essentially follow these requirements (should be made similar to the current jsx, copy to clipboard, and open in new window buttons):
https://dequeuniversity.com/rules/axe/3.1/button-name?application=lighthouse
